### PR TITLE
chore: add primary and secondary button styles

### PR DESCRIPTION
**Pull request by tujoworker**

### CODE REVIEW

File: packages/dnb-eufemia/src/components/button/__tests__/themes/Button.sbanken.screenshot.test.ts [hunk 1]
```diff
@@ -19,4 +19,66 @@ describe('Button with sbanken theme', () => {
...
+
+  it('have to match "dnb-button--secondary" with focus state', async () => {
+    const screenshot = await makeScreenshot({
+      selector: '[data-visual-test="button-secondary"]',
+      simulate: 'focus',
+    })
+    expect(screenshot).toMatchImageSnapshot()
+  })
+
+  // it('have to match disabled buttons', async () => {
```
Comment (langz) [inline packages/dnb-eufemia/src/components/button/__tests__/themes/Button.sbanken.screenshot.test.ts:78]: Should/do we need to include this out-commented test?

**Review by langz — APPROVED** [review]
Great test 🥇

**Comment by langz** [comment]
Is the image snapshots 100% correct?
It just seems a bit "jagged" or the border seems a bit "lower quality" some how 🤔 

Examples:
- https://raw.githubusercontent.com/dnbexperience/eufemia/29032d4af9c5e36813ce6f7a281a20eae6916b98/packages/dnb-eufemia/src/components/button/__tests__/themes/__image_snapshots__/button-with-sbanken-theme-have-to-match-dnb-button-secondary-with-hover-state.snap.png
- https://raw.githubusercontent.com/dnbexperience/eufemia/29032d4af9c5e36813ce6f7a281a20eae6916b98/packages/dnb-eufemia/src/components/button/__tests__/themes/__image_snapshots__/button-with-sbanken-theme-have-to-match-dnb-button-secondary.snap.png
-

**Comment by tujoworker** [comment]
> It's probably nothing we could/should change, and it's probably just the quality of our screenshot tests. Just an observation from me.

I'll try to find the reason. When I run `headless: false,` it looks very good in the browser that pops up 🤔

**Pull request merged**